x-pack/test/cloud_security_posture_functional/page_objects/findings_page.ts

```diff
@@ -52,50 +52,52 @@ export function FindingsPageProvider({ getService, getPageObjects }: FtrProvider
       testSubjects.click(type === 'failed' ? 'distribution_bar_failed' : 'distribution_bar_passed'),
   };
 
-  const table = {
-    getElement: () => testSubjects.find('findings_table'),
+  const latestFindingsTable = {
+    getElement: (selector: string) => testSubjects.find(selector),
 
-    getHeaders: async () => {
-      const element = await table.getElement();
+    getFindingsContainer: () => testSubjects.find('findings_container'),
+
+    getHeaders: async (selector: string) => {
+      const element = await latestFindingsTable.getElement(selector);
       return await element.findAllByCssSelector('thead tr :is(th,td)');
     },
 
     getColumnIndex: async (columnName: string) => {
-      const headers = await table.getHeaders();
+      const headers = await latestFindingsTable.getHeaders('findings_table');
       const texts = await Promise.all(headers.map((header) => header.getVisibleText()));
       const columnIndex = texts.findIndex((i) => i === columnName);
       expect(columnIndex).to.be.greaterThan(-1);
       return columnIndex + 1;
     },
 
     getColumnHeaderCell: async (columnName: string) => {
-      const headers = await table.getHeaders();
+      const headers = await latestFindingsTable.getHeaders('findings_table');
```

Comment:
`getHeaders` taking a table selector even though the method is on `latestFindingsTable` feels odd. the object should know its own table if we named it after it, instead of taking a random selector as an argument (in a lot of places)